ODRs/process_002_versioning_conventions.md

```diff
@@ -0,0 +1,74 @@
+## Versioning Conventions
+
+| ODR Info           | Details                            |
+|--------------------|------------------------------------|
+| Subject            | Versioning Conventions             |
+| ODR Number         | 002                                |
+| Status             | 	Proposed |
+| Category           | 	Process |
+| Author             | Dylan            |
+| Date               | 25.11.2024        |
+| Owner              | [Insert Owner/approver (Name)]    |
+| Review Date        | [Insert Last Review Date (DD.MM.YYYY)] |
+
+
+---
+
+## Context
+*To streamline development and release procedures, it is clear that a uniform versioning convention is required.*
+
+---
+
+## Decision
+All projects and packages inside the `views-platform` organization (e.g `views-pipeline-core` and `views-stepshifter`) will adopt the Semantic Versioning (SemVer) convention. You should do this by updating the `pyproject.toml` file for your respective package before submitting a release.
+
```

Comment:
_You should do this by updating the `pyproject.toml` file for your respective package before submitting a release._
We can automatise it:
- it can be published automatically when the .toml file is changed on the `main` branch with a GitHub action
- additionally, the release can also be submitted automatically via the GitHub workflow

The only guideline would be that whenever the .toml file changes on `main`, a new release will be submitted and the published.